composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "require": {
         "PHP": "^8.0",
-        "squizlabs/php_codesniffer": "^3.6",
+        "squizlabs/php_codesniffer": "3.6.0",
```

Comment:
Why was this needed? 👀

Reply:
CI was failing on `3.6.1`. Haven't had chance to investigate further

Reply:
Alright 👍 Think we need to figure this out before limiting it to that version.